README.md

```diff
@@ -98,7 +98,7 @@ Those are "date of the last update", "importance" and "assignee" (if there is an
 ### Further options and use cases of bug expiration
 
 The expiration is defined as 60 days of inactivity in server-next tagged bugs, and 180 days for the other ubuntu-server subscribed bugs.
-These durations as well as the tag it considers for the "active" list can be tuned via the arguments, --expire-tagged, --expire and --tag.
+These duration's as well as the tag it considers for the "active" list can be tuned via the arguments, --expire-tagged, --expire and --tag.
```

Comment:
Um, shouldn't it be `durations` only? :o

Reply:
Well aspell suggested that and I didn't think too deeply into it.
But you are right it isn't showing ownership (genitive) for sure so yes this is wrong.
/me stares at aspell and re-fixes it :-)